### PR TITLE
Feature/add query parameters

### PR DESCRIPTION
**Pull request by ashokpant**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -71,17 +71,17 @@
...
-<!--            <dependency>-->
-<!--                <groupId>org.springframework.ai</groupId>-->
-<!--                <artifactId>spring-ai-bom</artifactId>-->
-<!--                <version>${spring-ai.version}</version>-->
-<!--                <type>pom</type>-->
-<!--                <scope>import</scope>-->
-<!--            </dependency>-->
-<!--        </dependencies>-->
-<!--    </dependencyManagement>-->
+    <!--    <dependencyManagement>-->
```
Comment (springframeworkguru) [inline pom.xml:74]: Remove this, not needed

File: pom.xml [hunk 2]
```diff
@@ -215,6 +215,10 @@
             <artifactId>lombok</artifactId>
             <optional>true</optional>
         </dependency>
+        <dependency>
```
Comment (springframeworkguru) [inline pom.xml:218]: Remove netty, we're not using netty

File: src/main/java/org/jemberai/dataintake/service/QueryServiceImpl.java
Comment (springframeworkguru) [inline src/main/java/org/jemberai/dataintake/service/QueryServiceImpl.java:1]: Properties of Query Request need to be updated, OA3 specification needs to be updated

File: src/main/java/org/jemberai/dataintake/service/QueryServiceImpl.java [hunk 3]
```diff
@@ -56,20 +56,23 @@ public List<QueryResponseDocument> getDocuments(String clientId, QueryRequest qu
...
                 .toList();
 
         List<EventRecordChunk> chunks = eventRecordChunkRepository.findAllByEventRecord_ClientIdAndEmbeddingIdIn(clientId, matchIds);
 
         return chunks.stream().map(chunk -> QueryResponseDocument.builder()
-                .id(chunk.getId().toString())
-                .id(chunk.getEmbeddingId())
-                .content(new String(chunk.getData()))
-                .build())
+                        .id(chunk.getId().toString())
```
Comment (springframeworkguru) [inline src/main/java/org/jemberai/dataintake/service/QueryServiceImpl.java:73]: Add embedding id to query response document, update OA3 specification

Comment (springframeworkguru) [reply src/main/java/org/jemberai/dataintake/service/QueryServiceImpl.java:73]: Also populate metadata, add parent document id

**Review by springframeworkguru — CHANGES_REQUESTED** [review]
Please see comments and resolve

**Comment by ashokpant** [comment]
OA3 specification PR - https://github.com/jemberai/jember-oa3/pull/15

**Review by springframeworkguru — APPROVED** [review]

**Pull request merged**